.github/workflows/full-site-editing-plugin.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     - name: Install dependencies
       run: yarn install --frozen-lockfile
     - name: Build FSE
-      run: npx lerna run build --scope='@automattic/full-site-editing' --stream
+      run: cd apps/full-site-editing && yarn build
```

Comment:
Assuming that `yarn` comes pre-installed on github's image then this is fine. I assume this workflow gets tested as part of the PR.

Reply:
yeah, I'll make sure to trigger it. (as you can see, we are already using yarn elsewhere in the file, which was merged as part of yarn support in general)

Reply:
Oh, didn't spot that actually. All good then :D